WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -13,8 +13,16 @@ final class CardReaderSettingsViewController: UIViewController {
     private lazy var connectView = CardReaderSettingsConnectView()
     private lazy var connectedView = CardReaderSettingsConnectedReaderView()
 
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        /// We do not want to show the tabs on card reader settings screens
+        hidesBottomBarWhenPushed = true
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
+        /// Needed to avoid incorrect background appearing near bottom of view, especially on dark mode
+        view.backgroundColor = .systemBackground
```

Comment:
Not a blocker. Have you considered encapsulating this in a private method? Now there seems to be a bit of an imbalance, or asymmetry (everything else are methods)

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3950/commits/26b6425e46cb4523f3e2f99f1b078ab941be81fc